### PR TITLE
Added recommended parameters

### PR DESCRIPTION
**Pull request by ivanmkc**
Each dataset can set recommended parameters for model training. 

These will be set to the frontend as pre-selected options for the webapp.

### CODE REVIEW

File: services/dataset_service.py [hunk 1]
```diff
@@ -10,14 +10,28 @@
...
+        },
     ),
-    dataset.CSVDataset(
-        "sample_data/201306-citibike-tripdata.csv",
-        display_name="NYC Bike Traffic",
-        time_column="starttime",
-        description="This dataset includes bike traffic data from CityBikes",
-        icon="directions_bike",
-    ),
+    # dataset.CSVDataset(
```
Comment (ivanmkc) [inline services/dataset_service.py]: Removed this dataset and it's not a time-series dataset.

File: training_methods/bqml_training_method.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ def id(self) -> str:
         Returns:
             str: The id
         """
-        return "bqml"
+        return "bqml_arimaplus"
```
Comment (ivanmkc) [inline training_methods/bqml_training_method.py:20]: Switched to bqml_arimaplus

**Review by soheilazangeneh — APPROVED** [review]
How about the required parameters? How do the users know if some parameters are required? Are these labelled somehow on the UI as required/recommended? Or do the users get errors if missing a required one? I know in the backend we raise an error if they are missing!

**Comment by ivanmkc** [comment]
@soheilazangeneh Good question, currently the setup is a little strange as the definition of what parameters are available to the user is defined in the UI. However, the recommended parameters are defined in the backend.

Ideally, we'll refactor and move everything to the backend, essentially moving to a server-driven UI for the model training screen.

For errors, the error message from the training service (BQML or Vertex AI) will be passed to the user and displayed on the frontend. We can discuss this further when we decide to move to a more server-driven model.

**Pull request merged**